java/bulk-export/bulk-export-core/pom.xml

```diff
@@ -23,4 +23,48 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <artifactId>bulk-export-core</artifactId>
+
+    <dependencies>
+        <!-- Sleeper dependencies -->
+        <dependency>
+            <groupId>sleeper</groupId>  
+            <artifactId>core</artifactId>
+            <version>${project.parent.version}</version>   
+        </dependency>
+        <!-- Test dependencies -->
+        <dependency>
+            <groupId>sleeper</groupId>
+            <artifactId>core</artifactId>
+            <version>${project.parent.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
It looks like these test dependencies aren't needed for now.

Reply:
Removed